appendices/F/1_flip_int/go.mod

```
@@ -0,0 +1,3 @@
+module github.com/ablqk/tiny-go-projects/appendices/F
+
+go 1.22.4
```

```suggestion
go 1.23.0
```
(and subsequent files)